AndelaOSP/shufflebox-frontend
# Serviceworker

#### What does this PR do?
Add Service worker to emmitted bundle
#### Description of Task to be completed?
Add Service worker to the bundle emitted to provide an offline-first approach. And enabling the application to be delivered progressively( PWA). [Series](https://medium.com/@addyosmani/progressive-web-apps-with-react-js-part-i-introduction-50679aef2b12)
#### How should this be manually tested?
You can run `yarn build` / `npm build` and the application should emit a production build minified for consumption
